lib/commands/new_command.ts

```diff
@@ -116,3 +118,40 @@ async function getFileTextIfExists(path: string) {
     }
   }
 }
+
+async function checkIfRequiredToolsExist() {
+  const requiredTools = ["deno", "cargo", "rustup"];
+  const optionalTools = ["wasm-bindgen"];
```

Comment:
We can get rid of this. `wasm-bindgen` is not necessary for `wasmbuild`.

Reply:
(this part= the optional tool part not the code I linked)

Reply:
That’s only for when wasmbuild is building itself. It’s not used by consumers of wasmbuild.

Reply:
make sense, I updated the code